pkg/gatewayserver/io/udp/udp.go

```diff
@@ -61,6 +71,11 @@ var DefaultConfig = Config{
 	ConnectionExpires:   1 * time.Minute,  // Expire connection after missing typically 2 status messages.
 	ScheduleLateTime:    800 * time.Millisecond,
 	AddrChangeBlock:     1 * time.Minute, // Release address when the connection expires.
+	RateLimiting: RateLimitingConfig{
+		Enable:    true,
```

Comment:
We might not want to enable this by default

Reply:
We actually do